Sources/ClientRuntime/EventStream/DefaultMessageDecoderStream.swift

```diff
@@ -8,28 +8,36 @@
 extension EventStream {
 
     /// Stream adapter that decodes input data into `EventStream.Message` objects.
-    public struct DefaultMessageDecoderStream<Event: MessageUnmarshallable>: MessageDecoderStream {
+    public struct DefaultMessageDecoderStream<Event>: MessageDecoderStream {
```

Comment:
Converts `DefaultMessageDecoderStream` to use closures instead of `Swift.Decoder` to decode event payloads